Fail to start frontend page

<!-- 
Thank you for contributing to ML-Model_CI!
-->

#### Software and Hardware Versions 
ModelCI v1.x.x, CUDA Version vx.x.x, GPU device used...

#### Problem description

Here is the error shown on the browser
![Screenshot_2020-12-25 ML ModelCI](https://user-images.githubusercontent.com/35693452/103129760-f5748f80-46d4-11eb-93c1-ccd24f85a86d.png)

Here is the terminal log output
```
info WEBPACK Hash: a70f8812add50bc0dc61
info WEBPACK Version: webpack 4.43.0
info WEBPACK Child Client:
info WEBPACK     Hash: a70f8812add50bc0dc61
info WEBPACK     Time: 24455ms
info WEBPACK     Built at: 2020/12/25 下午5:14:35
info WEBPACK                 Asset       Size  Chunks                   Chunk Names
info WEBPACK         css/index.css   1.23 MiB   index  [emitted]        index
info WEBPACK     css/index.css.map      3 MiB   index  [emitted] [dev]  index
info WEBPACK           favicon.png   4.83 KiB          [emitted]        
info WEBPACK            index.html  567 bytes          [emitted]        
info WEBPACK           js/index.js   5.93 MiB   index  [emitted]        index
info WEBPACK       js/index.js.map   5.48 MiB   index  [emitted] [dev]  index
info WEBPACK     
info WEBPACK     ERROR in ./src/pages/Dashboard/index.tsx
info WEBPACK     Module not found: Error: Can't resolve 'reqwest' in '/home/sherry/project/github/ML-Model-CI/frontend/src/pages/Dashboard'
info WEBPACK      @ ./src/pages/Dashboard/index.tsx 40:0-30 126:6-13
info WEBPACK      @ ./.ice/pages/Dashboard/Dashboard.tsx
info WEBPACK      @ ./src/routes.ts
info WEBPACK      @ ./node_modules/build-plugin-ice-router/lib/module.js
info WEBPACK      @ ./.ice/createApp.tsx
info WEBPACK      @ ./.ice/index.ts
info WEBPACK      @ ./src/app.ts
info WEBPACK      @ multi ./node_modules/react-dev-utils/webpackHotDevClient.js ./src/app
info WEBPACK     Child html-webpack-plugin for "index.html":
info WEBPACK              Asset     Size  Chunks  Chunk Names
info WEBPACK         index.html  534 KiB       0  
(Client) ./src/pages/Dashboard/index.tsx
Module not found: Can't resolve 'reqwest' in '/home/sherry/project/github/ML-Model-CI/frontend/src/pages/Dashboard'

```

#### Steps to Reproduce the Problem

```shell
npm install
npm start
```

#### Expected Behavior
A webpage is shown on the browser.

#### Other Information 
Things you tried, stack traces, related issues, suggestions on how to fix it...

install reqwest package and add this dependency to `package.json` file will fix this 

```shell
npm i reqwest
```

--- a/frontend/package.json
+++ b/frontend/package.json
@@ -9,7 +9,8 @@
     "axios": "^0.19.2",
     "moment": "^2.24.0",
     "react": "^16.4.1",
-    "react-dom": "^16.4.1"
+    "react-dom": "^16.4.1",
+    "reqwest": "^2.0.5"
   },
   "devDependencies": {
     "@ice/spec": "^1.0.0",

